docs/EN_US/HPCC4J/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+################################################################################
+#    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems®.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+################################################################################
+get_filename_component(DOC_DIR_NAME ${CMAKE_CURRENT_SOURCE_DIR} NAME)
+include(${CMAKE_CURRENT_SOURCE_DIR}/../../BuildTools/cmake_config/${DOC_DIR_NAME}.txt)
```

Comment:
Minor, but why not continue using paths based on ${HPCC_SOURCE_DIR} rather than ${CMAKE_CURRENT_SOURCE_DIR} ?

IE.
include(${HPCC_SOURCE_DIR}/docs/BuildTools/cmake_config/${DOC_DIR_NAME}.txt)

Reply:
This conforms to the style consistent through all the EN_US and PT_BR Docs. Leaving stet.